### PR TITLE
fix(VSlider) add more aria attributes to VSlider

### LINKED ISSUE #4806: [Feature Request] V-slider a11y:  readonly is always set to true and aria-readonly is present which isn't allowed by specs
### Problem to solve
https://codepen.io/lol2x/pen/oMMmdx
Only the second slider have readonly prop set to true, however the HTML generated is always with readonly="readonly". 
What is more aria-readonly attribute should only be used for elements that are not allowed to have a readonly attribute in HTML5 - according to spec https://www.w3.org/TR/html-aria/

### Proposed solution
The presence of HTML readonly prop should be based on readonly prop from Vuetify
`readonly: String(this.readonly)`
and https://github.com/vuetifyjs/vuetify/blob/dev/src/components/VSlider/VSlider.js#L253 should be removed
I can create PR if you want this changes.

### PR DESCRIPTION
**Pull request by lol2x**
## Description

Aria-label was passed to root element without ...this.$attrs which causes errors in Google Insights
Also adds aria attributes -min -max -now

## Motivation and Context

fixes #4806

## How Has This Been Tested?

unit, playground

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-content>
      <v-container>
        <v-slider aria-label="just aria label"></v-slider>
        <v-slider label="this is label with aria label" aria-label="foo"></v-slider>
        <v-slider label="this is just label"></v-slider>
      </v-container>
    </v-content>
  </v-app>
</template>

<script>
  export default {
    data: () => ({
    })
  }
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have created a PR in the documentation with the necessary changes.

### CODE REVIEW

File: src/components/VSlider/VSlider.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export default {
   props: {
     alwaysDirty: Boolean,
     inverseLabel: Boolean,
+    hideLabel: Boolean,
```
Comment (KaelWD) [inline src/components/VSlider/VSlider.js]: This seems unnecessary, if you just add `...this.$attrs` to the bottom of the attrs object then the user can apply `aria-label` themselves.

Comment (KaelWD) [reply src/components/VSlider/VSlider.js]: - `label="something"` -> `<label>something</label>` + `aria-label="something"`
- `aria-label="something"` -> `aria-label="something"` + no `<label>`
- `label="something" aria-label="something else` -> `<label>something</label>` + `aria-label="something else"`

Comment (lol2x) [reply src/components/VSlider/VSlider.js]: will do

**Comment by lol2x** [comment]
fixed according to your suggestions, now this PR doesn't require change to docs, also fixed markup

**Pull request merged**